corehq/apps/reports/static/reports/js/bootstrap3/datatables_config.js

```diff
@@ -278,6 +281,65 @@ hqDefine("reports/js/bootstrap3/datatables_config", [
             });
         };  // end of self.render
 
+        self.renderTimingProfile = function (reportTimingProfile) {
+            var $timingProfile = $('#report-timing-profile');
+            if (!reportTimingProfile) {
+                $timingProfile.empty();
+                return;
+            }
+            // Create the main container
+            var html = `
+                <div class="panel panel-default">
+                  <div class="panel-heading">
+                    <h4 class="panel-title">
+                      Report Timing Profile:
+                      ${reportTimingProfile.name}
+                      (${reportTimingProfile.duration.toFixed(2)}s)
+                    </h4>
+                  </div>
+                  <div class="panel-body">
+                    ${renderTimingNode(reportTimingProfile)}
+                  </div>
+                </div>`;
```

Comment:
I'm curious to hear thoughts about this block of HTML, and the one below it. I considered using two templates instead, but that felt like it would be more complicated, and, on balance, defining the HTML here would be simpler to maintain.

Reply:
Thank you for the feedback. I'm happy to follow it up in a second PR.

To address the panel styles, I can change them to card styles in `reports/js/bootstrap5/datatables_config.js`. Or should I rather move these two blocks into templates?